hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/NSSummary.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+
+import org.apache.hadoop.ozone.recon.ReconConstants;
+
+import java.util.Arrays;
+
+/**
+ * Class to encapsulate namespace metadata summaries from OM.
+ */
+
+public class NSSummary {
+  private int numOfFiles;
+  private int sizeOfFiles;
+  private int[] fileSizeBucket;
+
+  public NSSummary() {
+    this.numOfFiles = 0;
+    this.sizeOfFiles = 0;
+    // TODO: I read the min is 1024(2^10), max is 1PB(2^50),
+    //  so the number of buckets should be 40?
+    this.fileSizeBucket = new int[ReconConstants.NUM_OF_BINS];
```

Comment:
Or, we can move `FileSizeCountTask.MAX_FILE_SIZE_UPPER_BOUND` to `ReconConstants` and calculate number of bins from it in `ReconConstants`.